pcp/Platform.c

```diff
@@ -501,6 +516,9 @@ void Platform_init(void) {
    Metric_enable(PCP_UNAME_MACHINE, true);
    Metric_enable(PCP_UNAME_DISTRO, true);
 
+   for(unsigned int i = pcp->columns.offset; i < pcp->columns.offset + pcp->columns.count; i++)
+      Metric_enable(i, true);
```

Comment:
Whitespace missing on 'for' - and this isn't a great approach in general.  We really only want to enable sampling of these metrics once they've been selected in the UI or read from htoprc (i.e. disabled by default, enabled only if used).  Otherwise we'll be sampling more and more metrics as more configurations are added, for no gain - only do it if needed.